src/pages/topics/Topic.tsx

```diff
@@ -64,6 +66,18 @@ const Topic: React.FC<Props> = ({ match }) => {
     setCases(cases);
     // @ts-ignore
     setReferences(references);
+
+    const constitution = new DOMParser().parseFromString(constitutionData.body, 'text/html');
```

Comment:
(a) we don't want to parse this every time the page changes. Instead, like we do in the constitution provision detail view, do this parsing outside of the react component.

(b) we're probably parsing this three times in various places now. Can you add a new `data/constitution.tsx` file that exports the parsed HTML so it can be used in various places throughout the app?